tests/test_cinoh_Board_of_Ed.py

```
@@ -0,0 +1,75 @@
+from datetime import datetime
+from os.path import dirname, join
+
+import pytest
+from city_scrapers_core.constants import COMMITTEE, NOT_CLASSIFIED, BOARD
```

Ruff (0.8.2)

`city_scrapers_core.constants.NOT_CLASSIFIED` imported but unused

Remove unused import

(F401)